### ISSUE TITLE
Revamp 03 | Rebuild main structure

### ISSUE THREAD

**Issue by SamusElderg** [spartan-protocol/SpartanProtocol-DAppV2#730]
- Rebuild main structure (header, body, nav, footer etc) using css-grid for better vertical control and layouts on each breakpoint
- Whilst there, reconsider the sidebar and header/navbar layouts/behaviours on each device size

This is a structural focus, not a style focus, get the structure responding to each screen size really well and don't worry about how the content looks in each section

When complete, still do not merge into main, this stays pending until at least part-way through the next revamp step

**Comment by SamusElderg** [comment]
we can consider this complete with f69add1e94e4756372b32fa83450cf6e4f9ce037
